packages/api/src/StyleOptions.ts

```diff
@@ -150,7 +150,28 @@ type StyleOptions = {
   bubbleFromUserNubSize?: number;
 
   bubbleFromUserTextColor?: string;
-  bubbleImageHeight?: number;
+
+  /**
+   * Specifies the fixed height of the bubble for image, default to unset.
+   *
+   * @deprecated Use `bubbleImageMaxHeight` and `bubbleImageMinHeight` instead. To mimick behavior before deprecation, set both options to 240px.
+   */
+  bubbleImageHeight?: number | undefined;
+
+  /**
+   * Specifies the maximum height of the bubble for image, default to 240px.
+   *
+   * New in 4.18.0.
+   */
+  bubbleImageMaxHeight?: number | undefined;
```

Comment:
Does it make sense to introduce these properties as deprecated from the start? This could:
- highlight our vision to migrate to CSS variables for stylistic changes
- communicate upcoming fundamental changes as early as possible
- provide more clear timeline for the upcoming change

Reply:
IMO, the "`styleOptions` -> CSS variables" feature won't deprecate `styleOptions` until few years later. It will remain as an overlapping feature for extended periods.

The roadmap is in this issue, https://github.com/microsoft/BotFramework-WebChat/issues/4850. But I agree we don't have a good way to list out all roadmaps.

I badly want to do the CSS variables.